tests/unit/test_launchers.py

```diff
@@ -55,6 +55,7 @@ def launcher_start(self, profile_class, *args, **kwargs):
 
     def test_installed(self):
         self.assertEqual(self.launcher.binary, '/binary')
+        del self.launcher
```

Comment:
I had to delete manually on each functions of this test case because the last test case tests that a call to __del__ actually delete the tempdir.